### PR TITLE
move troubleshooting to help screen

### LINKED ISSUE #514: Guides: Surface troubleshooting guide earlier
Considering the entry point to the site from the app is https://tachiyomi.org/help/, it should be clearer how to view the troubleshooting guide from there.

### PR DESCRIPTION
**Pull request by Seishirou101**
closes #514

Replaced Extensions Frequently Asked Question link to Troubleshooting in the Help Center

### CODE REVIEW

File: src/help/README.md [hunk 1]
```diff
@@ -9,10 +9,10 @@ help:
     description: Frequently asked questions
```
Comment (ghostbear) [inline src/help/README.md:9]: Maybe change the title of `Application` to `FAQ` or `Frequently asked questions`  and description to `About the application and extensions`

Comment (Seishirou101) [reply src/help/README.md]: I went with FAQ, but I feel like its looks slightly off with such a long description. Though was better than Frequently asked question though imo.

File: src/help/README.md [hunk 2]
```diff
@@ -5,14 +5,14 @@ description: From common questions to guides, find help for everything Tachiyomi
 lang: en-US
 hideSearch: true
 help:
-  - title: Application
-    description: Frequently asked questions
-    link: /help/faq/#application
+  - title: FAQ
+    description: About the application and extensions
```
Comment (ghostbear) [inline src/help/README.md:9]: > I went with FAQ, but I feel like its looks slightly off with such a long description. Though was better than Frequently asked question imo.

Could probably work with something shorter like `About app-related things` for the description

The icon should probably also be changed to `frequently-asked-questions`

Comment (Seishirou101) [reply src/help/README.md]: > Could probably work with something shorter like `About app-related things` for the description


That looks a lot better!

> The icon should probably also be changed to `frequently-asked-questions`

Done.

**Review by ghostbear — APPROVED** [review]
LGTM 👍

**Pull request merged**